octoprint_mrbeam/static/js/settings/dev_design_store.js

```diff
@@ -57,6 +59,17 @@ $(function () {
             self.showAuthToken();
         };
 
+        self.changeEnv = function () {
+            if (self.selectedEnv() === 'prod') {
+                self.designStore.DESIGN_STORE_IFRAME_SRC = 'https://designs.cloud.mr-beam.org';
```

Comment:
it's a good practice to define data like URLs as constants at the beginning of the file.